visidata/addons/pivot.py

```diff
@@ -3,18 +3,21 @@
 command('W', 'vd.push(SheetPivot(sheet, [cursorCol]))', 'push a sheet pivoted on the current column')
 
 class SheetPivot(Sheet):
+    """Summarize key columns in pivot table and display as new sheet."""
     def __init__(self, srcsheet, variableCols):
-        super().__init__(srcsheet.name + '_pivot', srcsheet)
+        super().__init__(srcsheet.name+'_pivot', srcsheet)
 
         self.nonpivotKeyCols = []
         self.variableCols = variableCols
         for colnum, col in enumerate(srcsheet.keyCols):
             if col not in variableCols:
-                newcol = Column(col.name, getter=lambda r,colnum=colnum: r[0][colnum])
+                newcol = Column(col.name, getter=lambda r, colnum=colnum: r[0][colnum])
                 newcol.srccol = col
                 self.nonpivotKeyCols.append(newcol)
 
-        self.command(ENTER, 'vd.push(source.copy(cursorCol.aggvalue)).rows = cursorRow[1].get(cursorCol.aggvalue, [])', 'push sheet of source rows aggregated in this cell')
+        self.command(ENTER, '''vd.push(source.copy(cursorCol.aggvalue)).rows='''
```

Comment:
I avoided proofreading of this sort, which is usual in Python to keep every line below 80 characters.

Custom is to use triple quotes for long strings (if the strings occur naturally within `()`, `[]`, or `{}` they will be treated as a single string).

Custom is to break _after_ operators, rather than before. So 

```python
self.command(ENTER, '''vd.push(source.copy(cursorCol.aggvalue))."""
                       """rows=...'''
```

or 

```python
self.command(ENTER, '''vd.push(source.copy(cursorCol.aggvalue)).rows="""
                       """...'''
```

but not

```python
self.command(ENTER, '''vd.push(source.copy(cursorCol.aggvalue)).rows"""
                       """=...'''
```

Also, the custom is not to fiddle excessively with perfect indentation. I usually use eight spaces for a continuing line, regardless of content.